### ISSUE TITLE
Add efficiency data for air furnaces

### ISSUE THREAD

**Issue by khaddad** [canmet-energy/btap_tasks#161]
### Description
Write a decription of the issue. If it is a bug, please list all steps and attach a sample file, or sample ruby code that exhibits the bug. 

NECB 2011, 2015, and 2017 have reference efficiency data for furnaces which need to be included in BTAP.  

### Approach
After reviewing the issue, decribe your solution. If including git hyperlinks, ensure hyperlinks to existing code are to a version so it does not change with updates to git. 

Create json data files for furnace efficiency.

### Testing Plan
Enter testing plan, if you are going to use an existing test or make a new one. 

Verify that furnace efficiency data is implemented in OpenStudio archetype files.

### Waiting On
If this task is dependent on another task or NREL. Include notes Here. 

### Repositories Involved
https://github.com/canmet-energy/btap

### Select the Project Code you will be charging to 2018-2019

| Client 	|WSBE  	|Name   	|Manager   	
|----------	|---	|---	|---	
|PERD   |P-002518.001.03	|Building Archetype	|Phylroy Lopez

**Comment by phylroy** [comment]
@khaddad  to review when he returns.
